selinux/98selinux-microos/selinux-microos-relabel.sh

```diff
@@ -48,24 +57,41 @@ rd_microos_relabel()
             ret=1
         fi
     done
+    if [ $ret -eq 0 ]; then
+        # Mount /var and /etc, need to be relabelled as well for booting.
+        for mp in /var /etc; do
+            if ! findmnt "${ROOT_SELINUX}${mp}" >/dev/null \
+              && findmnt -snrvo TARGET -F "${ROOT_SELINUX}/etc/fstab" "$mp" >/dev/null; then
```

Comment:
Please use long options for findmnt (also below in _selinux_autorelabel_generator_).

Reply:
Done